std_msgs/Header header
sensor_msgs/PointCloud2 combinedCloud
docking/Cluster[] clusters
